pkg/util/util.go

```diff
@@ -75,6 +75,8 @@ const (
 	gigabyte             = 1024 * 1024 * 1024
 	petabyte             = gigabyte * 1024 * 1024
 	obcMaxSizeUpperLimit = petabyte * 1023
+	// DisableDefaultTopologyConstraints is Annotation name for disabling default topology Constraints
+	DisableDefaultTopologyConstraints = "disableDefaultTopologyConstraints"
```

Comment:
Why here? It should be in the noobaa_types.go as it is part of our yaml api.
Also for best practices of annotations I believe we want to set it with a prefix to something like `noobaa.io/skip_topology_spread_constraints`

I prefer to say skip vs disable so that it's clear that we are not disabling the feature, but just not operating it from our operator.